UPGRADE.md

```diff
@@ -25,6 +26,14 @@ A new worker is added to the Procfile to work for the new queue. If you're hosti
 
 This change doesn't require any changes if you are compiling the stylesheets synchronously using the `rake sharetribe:generate_customization_stylesheets_immediately` command during the deployment. However, if you are compiling the stylesheets asynchronously using the `rake sharetribe:generate_customization_stylesheets` command, then you need to make sure that you have at least one worker working for the "css_compile" queue.
 
+
+React on Rails build environment is added with this change. This means that production environment needs to have node set up. With Heroku this can be set with ```heroku buildpacks:set heroku/nodejs```. For other environments - see [npm instructions](https://docs.npmjs.com/getting-started/installing-node), [nvm](https://github.com/creationix/nvm), or [n](https://github.com/tj/n). In addition, production environments should have ```NODE_ENV=production`` set.
```

Comment:
not production environment but build environment. Important difference because we do not require node for runtime environment, only for building.